src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngine.cs

```diff
@@ -46,60 +48,30 @@ public class RazorViewEngine : IRazorViewEngine
             IRazorPageFactoryProvider pageFactory,
             IRazorPageActivator pageActivator,
             HtmlEncoder htmlEncoder,
-            IOptions<RazorViewEngineOptions> optionsAccessor,
+            IOptions<RazorViewEngineOptions> options,
             ILoggerFactory loggerFactory)
         {
             _pageFactory = pageFactory;
             _pageActivator = pageActivator;
-            _viewLocationExpanders = optionsAccessor.Value.ViewLocationExpanders;
             _htmlEncoder = htmlEncoder;
             _logger = loggerFactory.CreateLogger<RazorViewEngine>();
             ViewLookupCache = new MemoryCache(new MemoryCacheOptions
             {
                 CompactOnMemoryPressure = false
             });
-        }
 
-        /// <summary>
-        /// Gets the locations where this instance of <see cref="RazorViewEngine"/> will search for views.
-        /// </summary>
-        /// <remarks>
-        /// The locations of the views returned from controllers that do not belong to an area.
-        /// Locations are composite format strings (see http://msdn.microsoft.com/en-us/library/txafckwd.aspx),
-        /// which contains following indexes:
-        /// {0} - Action Name
-        /// {1} - Controller Name
-        /// The values for these locations are case-sensitive on case-sensitive file systems.
-        /// For example, the view for the <c>Test</c> action of <c>HomeController</c> should be located at
-        /// <c>/Views/Home/Test.cshtml</c>. Locations such as <c>/views/home/test.cshtml</c> would not be discovered
-        /// </remarks>
-        public virtual IEnumerable<string> ViewLocationFormats { get; } = new[]
```

Comment:
So I thought this looked suspicious, but for a different reason. In MVC 5.x and earlier this property was *settable*, which provided good value (and is why I made a particular remark in the bug) - you could instantiate a view engine, set this property, and go along your merry way. But here it's read-only anyway, so its value of being even public is somewhat dubious, though it's perhaps nice for debugging. Now, it is *virtual*, which means a derived class can "set" it to return whatever it wants, but we're introducing a new way for that to happen anyway (via options) that a derived class can use just as easily.

So the question is: what real value does this property have, aside from perhaps debugging? (And that's somewhat of a dubious value as well.)